torch/_jit_internal.py

```diff
@@ -178,11 +178,12 @@ class FunctionModifiers(object):
 
 def export(fn):
```

Comment:
there is the question of why something in our public API is located in _jit_internal though…

Reply:
It's defined there so it can be used in torch (namely in `torch.nn`) without creating a circular dependency on `torch.jit`. It gets aliased to `torch.jit.export`